api/filter/accesslog.proto

```diff
@@ -285,8 +326,9 @@ message StreamAccessLogsMessage {
   // structured metadata and is a performance optimization.
   Identifier identifier = 1;
 
-  // A list of access logs.
-  repeated AccessLogEntry logs = 2;
+  // A list of access logs. Only a SINGLE type should be sent per access log stream.
```

Comment:
You could make this a `oneof` if you embed the `repeated` inside a wrapper message.

Reply:
The other alternative is to to do:

```
message Xs {
  repeated X x = 1;
}

message Ys {
  repeated Y y = 1;
}

oneof {
  Xs xs = 1;
  Ys ys = 2;
}
```